### ISSUE TITLE
I am working on snake game and got this error.

### ISSUE THREAD

**Issue by Hakanbkl** [PyCQA/pylint#7849]
### Bug description

![Ekran görüntüsü 2022-11-25 203953](https://user-images.githubusercontent.com/117858132/204034935-9992bf7b-5cf6-4e42-aa23-f780ce1b95f2.png)

### Command used

```shell
pylint a.py
```

### Pylint output

```shell
SyntaxError: invalid syntax
```

### Expected behavior

dont know

### Pylint version

```shell
pylint=2.15.5.
```

### OS / Environment

windows 10

**Comment by clavedeluna** [comment]
Thanks for the report. Could you please remove the image and include the code as text? That's easy to parse for us. Also, please provide the entire pylint trace error. We will try to reproduce.

**Comment by Pierre-Sassoulas** [comment]
This indentation of ``pygame.draw`` is wrong you need to move it 4 spaces to the right before the else. But this is something you should search on the web, pylint is just the messenger, this is not the right place to ask your question about python.
